uqcsbot/scripts/advent.py

```diff
@@ -0,0 +1,61 @@
+from uqcsbot import bot, Command
+from requests.exceptions import RequestException
+from typing import Dict, List
+import requests
+
+LEADERBOARD_URL = 'https://adventofcode.com/2018/leaderboard/private/view/'
+LEADERBOARD_CODE = '246889'
```

Comment:
Given the fact that the URL will have to be updated manually every year anyway (and you never use it on its own), is there any point in having these two as separate variables?